src/dsp_tools/xmllib/serialise/serialise_file_value.py

```diff
@@ -24,21 +26,30 @@ def serialise_file_value(file_value: AbstractFileValue) -> etree._Element:
         BaseError: in case of an unknown class
     """
     if isinstance(file_value, FileValue):
-        return _serialise_file_value(file_value, "bitstream")
+        return _serialise_file_value(file_value, authorship_id, "bitstream")
     elif isinstance(file_value, IIIFUri):
-        return _serialise_file_value(file_value, "iiif-uri")
+        return _serialise_file_value(file_value, authorship_id, "iiif-uri")
     else:
         raise BaseError(
             f"file_value must be either a FileValue or a IIIFUri, but you provided {file_value.__class__.__name__}"
         )
 
 
-def _serialise_file_value(value: AbstractFileValue, tag_name: str) -> etree._Element:
-    attribs = {}
-    if value.permissions != Permissions.PROJECT_SPECIFIC_PERMISSIONS:
-        attribs["permissions"] = value.permissions.value
+def _serialise_file_value(value: AbstractFileValue, authorship_id: str, tag_name: str) -> etree._Element:
+    attribs = _serialise_metadata(value.metadata, authorship_id)
     if is_string_like(value.comment):
         attribs["comment"] = str(value.comment)
     ele = etree.Element(f"{DASCH_SCHEMA}{tag_name}", attrib=attribs, nsmap=XML_NAMESPACE_MAP)
     ele.text = str(value.value)
     return ele
+
+
+def _serialise_metadata(metadata: Metadata, authorship_id: str) -> dict[str, str]:
+    attribs = {
+        "license": metadata.license,
+        "copyright-holder": metadata.copyright_holder,
+        "authorship-id": authorship_id,
```

Comment:
It took me a while to get this. Perhaps add a clarifying inline comment like this?

```suggestion
        "authorship-id": authorship_id,  # the authorship that is contained in the metadata is the raw form, but we need its ID
```